basics/ide_development_instance.md

```diff
@@ -0,0 +1,34 @@
+---
+title: IDE Development Instances - Settings, Caches, Logs, and Plugins
+---
+
+A JetBrains feature for developing plugins is to run or debug a plugin project from within a JetBrains IDE, e.g. IntelliJ IDEA. 
+[Selecting the **Run** menu](https://www.jetbrains.com/help/idea/running-and-debugging-plugins.html) will launch 
+a _Development Instance_ of the IDE with your plugin enabled. 
+
+This page describes where the settings, caches, logs, and plugins are located for a Development Instance of the IDE.
+This information is stored in a different location than for the
+[installed IDE itself](https://intellij-support.jetbrains.com/hc/en-us/articles/206544519-Directories-used-by-the-IDE-to-store-settings-caches-plugins-and-logs).
+Please review the documentation for the installed IDE to understand conventions about product names and versions.
+These conventions are used below.
+
+### Directory Locations for Development Instances of the IDE
```

Comment:
This covers the case when a plugin is built using DevKit. Please also document how to find the directories for plugins built with Gradle.